templates/erp/includes/fields.html

```diff
@@ -1,41 +1,31 @@
 {% load a4a %}
 {% load static %}
 {% load i18n %}
-{% if fields %}
-    <ul class="p-0 m-0">
-        {% for field in fields %}
-            {% render_field field.value as field_value %}
-            {% if field_value %}
-                <li class="a11y-field border-bottom">
-                    <!-- {{ field.name }} -->
-                    <p class="h6 m-0 p-0 font-weight-normal">
-                        {% if field.is_comment %}<i aria-hidden="true" class="icon icon-bubble"></i>{% endif %}
-                        {% if field.warning %}
-                            <span class="sr-only">{% translate "Attention :" %}</span>
+<ul class="p-0 m-0">
+    {% for field in fields %}
+        {% render_field field.value as field_value %}
+        <li class="a11y-field border-bottom">
+            <p class="h6 m-0 p-0 font-weight-normal">
+                {% if field.is_comment %}<i aria-hidden="true" class="icon icon-bubble"></i>{% endif %}
+                {{ field.label }}
+                {% if field.values and not field.is_comment %}
+                    :
+                    {% for value in field.values %}
+                        {% if value|startswith:"http" %}
```

Comment:
I am not sure why this should be allowed @mlvernay maybe URLs in free text / comments ? But then why adding the link to the page, this looks like a weird behavior in this case

Reply:
Can't be in free text because of `field.values and not field.is_comment`

Reply:
I added this in the new card I created in order to see what should be done in this case.